controller-norbac.jsonnet

```diff
@@ -32,6 +32,8 @@ local controllerContainer =
   container.command(["controller"]) +
   container.livenessProbe(controllerProbe) +
   container.readinessProbe(controllerProbe) +
+  container.mixin.securityContext.readOnlyRootFilesystem(true) +
+  container.mixin.securityContext.runAsUser(1001) +
```

Comment:
Urgh. ksonnet-lib beta.1 has a bunch of bugs.  You've typed two lines and you've encountered two of the bugs ;)

- the base securityContext object isn't created implicitly by `container.default()`
- it seems it expects `runAsNonRoot` to take an object, not a simple integer.

Here are the workarounds:
```javascript
  container.securityContext(k.core.v1.podSecurityContext.default()) +
  container.mixin.securityContext.readOnlyRootFilesystem(true) +
  container.mixin.securityContext.runAsNonRoot(true) +
  {securityContext+: {runAsUser: 1001}} +
```

(I've also included `container.mixin.securityContext.runAsNonRoot(true)` because we may as well assert that runAsUser is doing what we think)

(I'll move to ksonnet-lib beta.2 RSN, but separately to this PR)